erpnext/crm/doctype/availability_of_slots/availability_of_slots.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
What is the purpose of this doctype? If it is used to store the state of appointment pipeline, can it not be computed on the fly?

Reply:
It's used as a child table in `Appointment Booking Settings`. It's used for setting what time the time slots are available each day of the week.
<img width="922" alt="image" src="https://user-images.githubusercontent.com/6195660/64945201-b3339000-d88d-11e9-9b0d-61bf89f4fd35.png">

Reply:
could rename it to something better, originally named it this way because it was in the gameplan thread this way